changes/ee/fix-10201.en.md

```diff
@@ -0,0 +1 @@
+In TDengine, Remove the redundant table name from the SQL template.
```

Comment:
Isn't that _database name_, according to https://docs.tdengine.com/taos-sql/table/#create-table ?

Reply:
Yes, it is `database` not `table`.